LPS-71613 Increase coverage and fix error log

adolfopa opened the pull request.

adolfopa committed:
LPS-71613 Refactor validation and variant handling

adolfopa committed:
LPS-71613 No source should be generated when there are no conditions

adolfopa committed:
LPS-71613 Test behaviour with null configuration status

adolfopa committed:
LPS-71613 Enabling/disabling non existant config. should do nothing

adolfopa committed:
LPS-71613 Test that any extra plugins are preserved

adolfopa committed:
LPS-71613 Use model instead of ID

This was causing the teardown to fail in some tests; the adaptive
media is deleted asynchronously, but the file entry isn't, so this was
causing a race condition.

adolfopa committed:
LPS-71613 Autogenerated

liferay-continuous-integration commented:
Pull request test invoked at http://test-1-2.liferay.com/job/test-subrepository-acceptance-pullrequest(master).

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-adaptive-media/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-2.liferay.com/job/test-subrepository-acceptance-pullrequest(master)/37/">test-subrepository-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-2.liferay.com//userContent/jobs/test-subrepository-acceptance-pullrequest(master)/builds/37/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h1>Some tests FAILED.</h1><p>Build Time: 57 seconds 159 ms</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-adaptive-media/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/com-liferay-adaptive-media/commit/22ff0fffb89c99014313267d18cb040191400d0d">22ff0fffb89c99014313267d18cb040191400d0d</a></p><h4>Job Summary:</h4><ul><li><a href="https://test-1-2.liferay.com/job/test-subrepository-acceptance-pullrequest(master)/37/"><strike><strong>test-subrepository-acceptance-pullrequest(master)</strong></strike></a></li></ul><h5>For more details click <a href="https://test-1-2.liferay.com/userContent/jobs/test-subrepository-acceptance-pullrequest(master)/builds/37/jenkins-report.html">here</a>.</h5><hr/><h4>Failed Jobs:</h4><ol><li><div><h5><a href="https://test-1-2.liferay.com/job/test-subrepository-acceptance-pullrequest(master)/37/">test-subrepository-acceptance-pullrequest(master)</a></h5><div><h6>Job Results:</h6><p>0 Jobs Passed.<br/>1 Job Failed.</p></div><pre><code>	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
	at org.apache.tools.ant.Task.perform(Task.java:348)
	at org.apache.tools.ant.taskdefs.MacroInstance.execute(MacroInstance.java:396)
	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
	at org.apache.tools.ant.Task.perform(Task.java:348)
	at org.apache.tools.ant.Target.execute(Target.java:435)
	at org.apache.tools.ant.Target.performTasks(Target.java:456)
	at org.apache.tools.ant.Project.executeSortedTargets(Project.java:1393)
	at org.apache.tools.ant.helper.SingleCheckExecutor.executeTargets(SingleCheckExecutor.java:38)
	at org.apache.tools.ant.Project.executeTargets(Project.java:1248)
	at org.apache.tools.ant.taskdefs.Ant.execute(Ant.java:441)
	at org.apache.tools.ant.taskdefs.CallTarget.execute(CallTarget.java:105)
	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
	at org.apache.tools.ant.Task.perform(Task.java:348)
	at org.apache.tools.ant.Target.execute(Target.java:435)
	at org.apache.tools.ant.Target.performTasks(Target.java:456)
	at org.apache.tools.ant.Project.executeSortedTargets(Project.java:1393)
	at org.apache.tools.ant.Project.executeTarget(Project.java:1364)
	at org.apache.tools.ant.helper.DefaultExecutor.executeTargets(DefaultExecutor.java:41)
	at org.apache.tools.ant.Project.executeTargets(Project.java:1248)
	at org.apache.tools.ant.Main.runBuild(Main.java:851)
	at org.apache.tools.ant.Main.startAnt(Main.java:235)
	at org.apache.tools.ant.launch.Launcher.run(Launcher.java:280)
	at org.apache.tools.ant.launch.Launcher.main(Launcher.java:109)</code></pre></div></li></ol></html>

sharonchoi committed:
LRQA-31670 Add lfrbuild-portal files

adolfopa commented:
ci:retest

liferay-continuous-integration commented:
Pull request test invoked at http://test-1-17.liferay.com/job/test-subrepository-acceptance-pullrequest(master).

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-adaptive-media/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-17.liferay.com/job/test-subrepository-acceptance-pullrequest(master)/41/">test-subrepository-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-17.liferay.com//userContent/jobs/test-subrepository-acceptance-pullrequest(master)/builds/41/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Validation PASSED. Running batch tests.</h1><p>Build Time: 10 minutes 40 seconds</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-adaptive-media/tree/master">master</a><br />Branch GIT ID: <a href="https://github.com/liferay/com-liferay-adaptive-media/commit/22ff0fffb89c99014313267d18cb040191400d0d">22ff0fffb89c99014313267d18cb040191400d0d</a></p><h4>Task Summary:</h4><ul><li><strong><a href="https://test-1-17.liferay.com//userContent/jobs/test-subrepository-acceptance-pullrequest(master)/builds/41/subrepository-compile-jdk8.log">subrepository-compile-jdk8</a></strong> -  :white_check_mark:</li><li><strong><a href="https://test-1-17.liferay.com//userContent/jobs/test-subrepository-acceptance-pullrequest(master)/builds/41/subrepository-semantic-versioning-jdk8.log">subrepository-semantic-versioning-jdk8</a></strong> -  :white_check_mark:</li><li><strong><a href="https://test-1-17.liferay.com//userContent/jobs/test-subrepository-acceptance-pullrequest(master)/builds/41/subrepository-pmd-jdk8.log">subrepository-pmd-jdk8</a></strong> -  :white_check_mark:</li><li><strong><a href="https://test-1-17.liferay.com//userContent/jobs/test-subrepository-acceptance-pullrequest(master)/builds/41/subrepository-source-format-jdk8.log">subrepository-source-format-jdk8</a></strong> -  :white_check_mark:</li><li><strong><a href="https://test-1-17.liferay.com//userContent/jobs/test-subrepository-acceptance-pullrequest(master)/builds/41/subrepository-unit-jdk8.log">subrepository-unit-jdk8</a></strong> -  :white_check_mark:</li></ul><h5>For more details click <a href="https://test-1-18.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(master)/47/">here</a>.</h5>

liferay-continuous-integration commented:
<html><h1>Some tests FAILED.</h1><p>Build Time: 1 hour 8 minutes 27 seconds 992 ms</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-adaptive-media/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/com-liferay-adaptive-media/commit/22ff0fffb89c99014313267d18cb040191400d0d">22ff0fffb89c99014313267d18cb040191400d0d</a></p><h4>Job Summary:</h4><ul><li><a href="https://test-1-17.liferay.com/job/test-subrepository-acceptance-pullrequest(master)/41/"><strike><strong>test-subrepository-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-4.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(master)/121/">test-subrepository-acceptance-pullrequest-batch(master)/functional-smoke-tomcat80-mysql56-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(master)/123/"><strike><strong>test-subrepository-acceptance-pullrequest-batch(master)/subrepository-functional-tomcat80-mysql56-jdk8/0</strong></strike></a></li><li><a href="https://test-1-2.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(master)/98/"><strike><strong>test-subrepository-acceptance-pullrequest-batch(master)/subrepository-integration-mysql56-jdk8</strong></strike></a></li><li><a href="https://test-1-18.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(master)/47/">test-subrepository-acceptance-pullrequest-validation(master)</a></li></ul><h5>For more details click <a href="https://test-1-17.liferay.com/userContent/jobs/test-subrepository-acceptance-pullrequest(master)/builds/41/jenkins-report.html">here</a>.</h5><hr/><h4>Failed Jobs:</h4><ol><li><div><h5><a href="https://test-1-17.liferay.com/job/test-subrepository-acceptance-pullrequest(master)/41/">test-subrepository-acceptance-pullrequest(master)</a></h5><div><h6>Job Results:</h6><p>2 Jobs Passed.<br/>3 Jobs Failed.</p></div><pre><code>Downstream jobs FAILED.</code></pre></div></li><li><div><h5><a href="https://test-1-4.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(master)/123/">test-subrepository-acceptance-pullrequest-batch(master)/subrepository-functional-tomcat80-mysql56-jdk8/0</a></h5><div><h6>Job Results:</h6><p>2 Tests Passed.<br/>4 Tests Failed.</p></div><ol><li><div><a href="https://test-1-4.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/123/">AXIS_VARIABLE=0,label_exp=!master #123</a><ol><li><a href="https://test-1-4.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/123//testReport/com.liferay.poshi.runner/PoshiRunner/test_AdaptiveMedia_AddAMIWithHighDensityResolution_">AdaptiveMedia#AddAMIWithHighDensityResolution</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-17/1491297473935/test-subrepository-acceptance-pullrequest(master)/41/subrepository-functional-tomcat80-mysql56-jdk8/0/0/AdaptiveMedia_AddAMIWithHighDensityResolution/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-17/1491297473935/test-subrepository-acceptance-pullrequest(master)/41/subrepository-functional-tomcat80-mysql56-jdk8/0/0/AdaptiveMedia_AddAMIWithHighDensityResolution/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-17/1491297473935/test-subrepository-acceptance-pullrequest(master)/41/subrepository-functional-tomcat80-mysql56-jdk8/0/0/jenkins-console.txt.gz">Console Output</a></li><li><a href="https://test-1-4.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/123//testReport/com.liferay.poshi.runner/PoshiRunner/test_AdaptiveMedia_AddImageVariantsForBlogEntry_">AdaptiveMedia#AddImageVariantsForBlogEntry</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-17/1491297473935/test-subrepository-acceptance-pullrequest(master)/41/subrepository-functional-tomcat80-mysql56-jdk8/0/0/AdaptiveMedia_AddImageVariantsForBlogEntry/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-17/1491297473935/test-subrepository-acceptance-pullrequest(master)/41/subrepository-functional-tomcat80-mysql56-jdk8/0/0/AdaptiveMedia_AddImageVariantsForBlogEntry/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-17/1491297473935/test-subrepository-acceptance-pullrequest(master)/41/subrepository-functional-tomcat80-mysql56-jdk8/0/0/jenkins-console.txt.gz">Console Output</a></li><li><a href="https://test-1-4.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/123//testReport/com.liferay.poshi.runner/PoshiRunner/test_AdaptiveMedia_EditImageResolutionWithOptimizedImages_">AdaptiveMedia#EditImageResolutionWithOptimizedImages</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-17/1491297473935/test-subrepository-acceptance-pullrequest(master)/41/subrepository-functional-tomcat80-mysql56-jdk8/0/0/AdaptiveMedia_EditImageResolutionWithOptimizedImages/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-17/1491297473935/test-subrepository-acceptance-pullrequest(master)/41/subrepository-functional-tomcat80-mysql56-jdk8/0/0/AdaptiveMedia_EditImageResolutionWithOptimizedImages/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-17/1491297473935/test-subrepository-acceptance-pullrequest(master)/41/subrepository-functional-tomcat80-mysql56-jdk8/0/0/jenkins-console.txt.gz">Console Output</a></li></ol></div></li><li><div><a href="https://test-1-4.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=1,label_exp=!master/123/">AXIS_VARIABLE=1,label_exp=!master #123</a><ol><li><a href="https://test-1-4.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=1,label_exp=!master/123//testReport/com.liferay.poshi.runner/PoshiRunner/test_AdaptiveMedia_EditImageResolutionWithoutOptimizedImage_">AdaptiveMedia#EditImageResolutionWithoutOptimizedImage</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-17/1491297473935/test-subrepository-acceptance-pullrequest(master)/41/subrepository-functional-tomcat80-mysql56-jdk8/0/1/AdaptiveMedia_EditImageResolutionWithoutOptimizedImage/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-17/1491297473935/test-subrepository-acceptance-pullrequest(master)/41/subrepository-functional-tomcat80-mysql56-jdk8/0/1/AdaptiveMedia_EditImageResolutionWithoutOptimizedImage/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-17/1491297473935/test-subrepository-acceptance-pullrequest(master)/41/subrepository-functional-tomcat80-mysql56-jdk8/0/1/jenkins-console.txt.gz">Console Output</a></li></ol></div></li></ol></div></li><li><div><h5><a href="https://test-1-2.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(master)/98/">test-subrepository-acceptance-pullrequest-batch(master)/subrepository-integration-mysql56-jdk8</a></h5><div><h6>Job Results:</h6><p>0 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-2.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/98/">AXIS_VARIABLE=0,label_exp=!master #98</a><pre><code>     [exec]         at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:54)
     [exec]         at org.gradle.internal.concurrent.StoppableExecutorImpl$1.run(StoppableExecutorImpl.java:40)
     [exec]         at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
     [exec]         at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
     [exec]         at java.lang.Thread.run(Thread.java:745)
     [exec] 
     [exec]         Caused by:
     [exec]         java.util.concurrent.TimeoutException
     [exec]             at java.util.concurrent.FutureTask.get(FutureTask.java:205)
     [exec]             at org.jboss.arquillian.container.osgi.jmx.JMXDeployableContainer.getMBeanServerConnection(JMXDeployableContainer.java:355)
     [exec]             at org.jboss.arquillian.container.osgi.karaf.remote.KarafRemoteDeployableContainer.start(KarafRemoteDeployableContainer.java:74)
     [exec]             ... 81 more
     [exec] :apps:adaptive-media:adaptive-media-document-library-thumbnails-test:testIntegration FAILED
     [exec] :apps:adaptive-media:adaptive-media-document-library-thumbnails-test:stopTestableTomcat SKIPPED
     [exec] 
     [exec] FAILURE: Build failed with an exception.
     [exec] 
     [exec] * What went wrong:
     [exec] Execution failed for task ':apps:adaptive-media:adaptive-media-document-library-thumbnails-test:testIntegration'.
     [exec] &gt; There were failing tests. See the report at: file:///opt/dev/projects/github/liferay-portal-ee-7.0.x/modules/apps/adaptive-media/adaptive-media-document-library-thumbnails-test/build/reports/tests/testIntegration/index.html
     [exec] 
     [exec] * Try:
     [exec] Run with --info or --debug option to get more log output.
     [exec] 
     [exec] * Exception is:
     [exec] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':apps:adaptive-media:adaptive-media-document-library-thumbnails-test:testIntegration'.
     [exec] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:84)
     [exec] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:55)
     [exec] 	at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:62)</code></pre></div></li></ol></div></li></ol></html>

ambrinchaudhary committed:
LPS-71712 Do not start progress if there are no images to optimize

4lejandrito committed:
LPS-71507 Always serving an image for an existing file entry

If the adaptive media image has not yet been generated the closest one
will be served. If no adaptive media images are present, the real
image will be used.

4lejandrito committed:
LPS-71507 Actually getting the closest adaptive media comparing widths

ambrinchaudhary committed:
LPS-71701 Set min value

adolfopa committed:
LPS-71507 Use a more concrete exception

adolfopa committed:
LPS-71507 Simplify control flow

adolfopa committed:
LPS-71507 Use Comparator.comparingInt

adolfopa committed:
LPS-71507 SF - Avoid chaining

4lejandrito committed:
LPS-70954 Dynamically replacing journal content

4lejandrito committed:
LPS-70954 Adding support for adaptive media in the web content editor

4lejandrito committed:
LPS-70954 Adding tests for the transformer listener

4lejandrito committed:
LPS-70229 Handling null inputs

4lejandrito committed:
LPS-71887 SF

adolfopa committed:
LPS-70954 SF
